pkg/testhelper/testhelper.go

```diff
@@ -214,6 +219,15 @@ func NewContainerReportObject(aNamespace, aPodName, aContainerName, aReason stri
 	return out
 }
 
+func NewCertifiedContainerReportObject(cii configuration.ContainerImageIdentifier, aReason string, isCompliant bool) (out *ReportObject) {
+	out = NewReportObject(aReason, ContainerType, isCompliant)
```

Comment:
It may be better to use a new ContainerImageType.

Reply:
Good idea, I'll add that.

Reply:
Done.